easybuild/easyblocks/generic/conda.py

```diff
@@ -79,7 +83,6 @@ def install_step(self):
 
             self.set_conda_env()
 
-            # use --force to ignore existing installation directory
             cmd = "%s conda env create --force %s -p %s" % (self.cfg['preinstallopts'], env_spec, self.installdir)
```

Comment:
Maybe we should have a separate option for conda options?

```
conda_options=['--force', '--copy']
```

There are a lot of options in conda.

FYI, I think I initially thought that --force would make conda behave nicely with an existing directory, but that does not see to be the case.